project1/styles.css

```diff
@@ -1,5 +1,67 @@
+body {
+    background-color: hsl(254, 0%, 100%);
+    border: 3px;
+}
+
+figure {
+    float: right;
+    margin-right: 40px;
+    padding: 30px;
+
+
+}
+
+iframe {
+    margin: 60px;
+}
+
 img {
-    max-width: 100%
+    max-width: 100%;
 }
 
-/* comments look like this */
+figcaption {
+    color: hsl(0, 0%, 40%);
+    font-family: 'amiri', serif;
```

Comment:
Note that you call a very specific font here, but if a user doesn't have the font on their computer, they'll just get a generic serif font. Same issue when you reference Quicksand later.

You should add your fonts from [Google Fonts](https://fonts.google.com/) and reference their stylesheet in your `index.html` so more users get the font you intend.